esmvaltool/utils/recipe_test_workflow/doc/source/user_guide/adding_a_recipe.rst

```diff
@@ -0,0 +1,100 @@
+Adding a recipe to the Recipe Test Workflow (|RTW|)
+===================================================
+
+.. include:: ../common.txt
+
+**Please note:** Before you follow these steps to add your recipe, you must be
+able to successfully run it with ESMValtool without error.
+
+* Run ESMValTool locally with your recipe, make sure to take note of the memory
+  and time expenditure provided by the terminal output::
+
+    esmvaltool run <your_recipe_name.yml>
+
+* Run the same recipe on JASMIN, again taking note of the memory and time
+  expenditure printed to the terminal.
+
+* Stop any running recipe_test_workflow workflows::
+
+    cylc stop "a_running_recipe_test_workflow"
+
+* Add the recipe to the ``[task parameters]`` section of the |RTW| ``flow.cylc``
+  workflow file (make sure to place it within the "fast" or "medium" category
+  depending on how long it took to run):
```

Comment:
Formatting the last row like this `recipe_new_recipe \,` seems to throw an error.

```bash
IllegalValueError: (type=parameter) [task parameters]fast = recipe_radiation_budget,         recipe_albedolandcover,         recipe_heatwaves_coldwaves,         recipe_autoassess_landsurface_soilmoisture,         recipe_consecdrydays,         examples--recipe_python,         recipe_ocean_multimap,     medium = recipe_ensclus  # Should be in fast: is in medium temporarily to - (medium = recipe_ensclus: bad value)


```

Reply:
> > * we also discussed adding the "slow" recipe option and criteria in the docs
> > * and adding a slow recipe to the `flow.cylc` so the section exists and isn't empty, to help guide a user placing a recipe there
> 
> This will be added once Emma has picked a slow recipe to use

I'm going to suggest this work is done via #3570.